pkg/providers/instancetype/suite_test.go

```diff
@@ -1512,6 +1512,7 @@ var _ = Describe("InstanceType Provider", func() {
 			// Well Known to AKS
 			{Name: v1beta1.LabelSKUName, Label: v1beta1.LabelSKUName, ValueFunc: func() string { return "Standard_NC24ads_A100_v4" }, ExpectedInKubeletLabels: true, ExpectedOnNode: true},
 			{Name: v1beta1.LabelSKUFamily, Label: v1beta1.LabelSKUFamily, ValueFunc: func() string { return "N" }, ExpectedInKubeletLabels: true, ExpectedOnNode: true},
+			{Name: v1beta1.LabelSKUSeries, Label: v1beta1.LabelSKUSeries, ValueFunc: func() string { return "NCads_v4" }, ExpectedInKubeletLabels: true, ExpectedOnNode: true},
```

Comment:
You'll need to add an entry to the `nodeSelector` in test "should support well-known labels for instance type selection" in test/suites/scheduling/suite_test.go as well.

I think you should be able to run that test locally if you `make az-all` and then execute the test. If you can do that can you share the results showing it's passing? Right now we don't have a great way to run E2E on forks, I think (on my list to fix but haven't gotten to it yet).

If you can't do that, let me know and I can try to run the test locally myself once you've made the change to ensure it passes.